app/views/employers/employer_attestations/verify_attestation.js.erb

```diff
@@ -12,6 +12,14 @@ $("input[name='status']").click(function(){
         $('.rejection_reason').removeClass('hidden');
         $('#attestation_submit').addClass('disabled');
         $("input[name='other_reason']").val('');
+        $("input[name='other_reason']").addClass('hidden');
+        if ($(this).val() === 'rejected') {
+          $("select[name=reason_for_rejection] option:first").text("Reason for rejection");
```

Comment:
Drop down menu header should be camelcase

Reply:
I did as it was mentioned on the ticket.